pepper-apis/ddp-common/src/main/java/org/broadinstitute/ddp/util/LiquibaseUtil.java

```diff
@@ -133,9 +131,6 @@ private void runDSMGlobalMigrations() throws LiquibaseException, SQLException {
     private void runMigrations(String changelogFile) throws LiquibaseException, SQLException {
         Liquibase liquibase = null;
         String tag = null;
-        if (ServiceLocator.getInstance() == null) {
```

Comment:
I'm hoping someone can help validate that this is not needed.  Perhaps this was put in for concurrent testing and we'll only see if it works when things run in CircleCI?

Reply:
In the updated libraries, I couldn't find a way to do something equivalent (same goes for L93).